src/main/java/gregtech/common/items/behaviors/TurbineRotorBehavior2.java

```diff
@@ -0,0 +1,77 @@
+package gregtech.common.items.behaviors;
+
+import gregtech.api.items.metaitem.MetaItem;
+import gregtech.api.items.metaitem.stats.IItemDurabilityManager;
+import gregtech.api.items.metaitem.stats.IItemMaxStackSizeProvider;
+import gregtech.api.unification.material.Material;
+import gregtech.api.unification.material.properties.PropertyKey;
+import gregtech.api.unification.material.properties.ToolProperty;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.item.ItemStack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class TurbineRotorBehavior2 extends AbstractMaterialPartBehavior implements IItemMaxStackSizeProvider {
+
+    private static final int TOOL_DURABILITY_MULTIPLIER = 100;
+
+    @Override
+    public int getPartMaxDurability(ItemStack itemStack) {
+        Material material = getPartMaterial(itemStack);
+        ToolProperty property = material.getProperty(PropertyKey.TOOL);
+        return property != null ? property.getToolDurability() * TOOL_DURABILITY_MULTIPLIER : 1;
```

Comment:
This is the only method that returns 1 instead of -1 for invalid passed itemstack. The behavior should be unified